README.md

```diff
@@ -949,6 +982,62 @@ again in a future request, it will use it directly from Nexus rather than downlo
 uploading it again. This guarantees that any dependency used for a Cachito request can be used again
 in a future Cachito request.
 
+### RubyGems (Bundler)
+
+The Bundler package manager works by parsing the `Gemfile.lock` file present in the source
+repository to determine what dependencies are required to build the application.
+
+Cachito then creates a RubyGems repository in an instance of Nexus it manages that contains just the
+GEM dependencies discovered in the lock file. Also, Cachito produces a bundle downloadable at
+`/api/v1/requests/<id>/download` containing `app/` directory with the application source code 
+(including PATH dependencies) and `/deps/rubygems` directory with all GEM and GIT dependencies.
+
+Since multiple packages in a single repo are supported, for each of these packages a configuration 
+file is provided at `/api/v1/requests/<id>/configuration-files` endpoint. This file redirects 
+Bundler to use Nexus proxy for downloading GEM dependencies and contains an entry for every Git 
+dependency to be overridden by the corresponding dependency from `deps/rubygems` (instead of 
+downloading it from the internet, see [local Git 
+repos](https://bundler.io/man/bundle-config.1.html#LOCAL-GIT-REPOS) for more details). 
+If a GIT dependency is specified with `branch:` in the Gemfile, this branch is checked out so that 
+local GIT repo redirection works.
+
+Note that configuration files expose the username and password of the temporary user created for 
+your request. This should not be a security concern, the user only has read access for 
+the repositories and the only reason why we do not allow anonymous read access is due to a technical
+limitation in Nexus.
+
+### Requirements for RubyGems repos
+
+There are several constraints on RubyGems packages that are enforced by Cachito and not meeting them
+raises an exception sooner or later:
+
+- To prevent Cachito from downloading native content (binaries) `Gemfile.lock` has to contain only
+  one platform in it's `PLATFORM` section and it has to be `ruby`.
+- All `PATH` dependencies listed in `Gemfile.lock` have to be explicitly allowed in Cachito's
+  config. For example, a repository at URL `github.com/cachito-testing/cachito-rubygems-multiple` with
+  `PATH` dependency `pathgem` will be processed properly only if Cachito's config contains the
+  following entry
+
+``` 
+cachito_rubygems_file_deps_allowlist = {
+    "cachito-rubygems-multiple/first_pkg": ["pathgem"] 
```

Comment:
It might be good to explicitly mention that the package is at the `first_pkg` subpath in the repo - otherwise it's not clear why a repo called "cachito-rubygems-multiple" needs an entry named "cachito-rubygems-multiple/first_pkg" (unless the reader can piece it together from the paragraph about `/subpath` below)